indico/modules/events/contributions/models/fields.py

```diff
@@ -44,6 +44,10 @@ class ContributionField(db.Model):
         index=True,
         nullable=False
     )
+    legacy_id = db.Column(
+        db.String,
+        nullable=True
+    )
```

Comment:
Maybe we can leave that for next version? It's quite a bit of work right now :disappointed_relieved: